tests/test-utils.js

```diff
@@ -3,11 +3,16 @@ import { createLocalVue, shallowMount, mount } from '@vue/test-utils'
 import Vuex from 'vuex'
 import { storeRaw } from '@/store'
 import Vuetify from 'vuetify'
-import wait from 'waait'
 
 const removeExtraSpaces = str => str.replace(/  +/g, ' ')
 const removeTabs = str => str.replace('\t', '')
 export const cleanString = str => removeTabs(removeExtraSpaces(str)).trim()
+export const wait = ms => {
+  if (process.env.CI !== "1" || process.env.INTEGRATION == "1") 
+    return new Promise(r => setTimeout(r, ms))
+  console.log("In CI so not going to wait");
+  return
+}
```

Comment:
why don't we need the waits in CI unit tests? (but we keep in them in CI integration tests?)

Not blocking, but I would find this logic easier to read as

```suggestion
export const wait = ms => {
  if (process.env.CI === "1" && process.env.INTEGRATION !== "1") {
    console.log("In CI so not going to wait");
    return
  }
  return new Promise(r => setTimeout(r, ms))
}
```

Might just be the way my mind works, but the `&&` requires less thinking for me than the equivalent `||`